Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/light_rifle.yml

```
@@ -8,7 +8,7 @@
     tags:
     - Cartridge
     - CartridgeLightRifle
-  - type: HitScanCartridgeAmmo
+  - type: HitScanCartridgeAmmo #Sunrise-Edit
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)